lib/streamlit/runtime/runtime.py

```diff
@@ -602,13 +602,16 @@ async def _loop_coroutine(self) -> None:
                 if self._state == RuntimeState.NO_SESSIONS_CONNECTED:  # type: ignore[comparison-overlap]
                     # mypy 1.4 incorrectly thinks this if-clause is unreachable,
                     # because it thinks self._state must be INITIAL | ONE_OR_MORE_SESSIONS_CONNECTED.
-                    await asyncio.wait(  # type: ignore[unreachable]
```

Comment:
Do we need both the contents of this `if` block as well as the code on L640?

It sees to me like we could replace this with a no-op and rely on the code below to do this since the code looks the same to me

Reply:
yes indeed, good point 👍  I think this is something we can refactor out. I replaced it with a pass.